datalad/utils.py

```diff
@@ -81,10 +81,19 @@
     linux_distribution_name = linux_distribution_release = None
 
 # Maximal length of cmdline string
-# Did not find anything in Python which could tell at run time and
+# Query the system and use hardcoded "knowledge" if None
 # probably   getconf ARG_MAX   might not be available
 # The last one would be the most conservative/Windows
-CMD_MAX_ARG = 2097152 if on_linux else 262144 if on_osx else 32767
+CMD_MAX_ARG_HARDCODED = 2097152 if on_linux else 262144 if on_osx else 32767
+try:
+    CMD_MAX_ARG = os.sysconf('SC_ARG_MAX') or CMD_MAX_ARG_HARDCODED
```

Comment:
Hmm, based on quick googling, I think `os.sysconf('SC_ARG_MAX')` will only work on POSIX systems.

Reply:
good that I have try/except then already ;)

Reply:
> good that I have try/except then already ;)

Yes, and then in the except clause you say:

    # yoh: do not know when it might/will fail but I would prefer to play
    # safe while querying `sysconf`

That seems like a confusing comment if we expect it to fail on Windows.

Reply:
got it... I will adjust that as of today on Windows it would fail